lab_1_classify_by_unigrams/main.py

```diff
@@ -37,6 +39,17 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     :param actual: a list of actual values
     :return: the score
     """
+    count_actual = len(actual)
+    count_predicted = len(predicted)
+    summ_values = 0
+    if isinstance(actual, list) and isinstance(predicted, list) and count_actual == count_predicted:
```

Comment:
again as in the first function - simplify if/else statement

Reply:
Fixed